### PR TITLE
Fix event card responsiveness for tags.

### LINKED ISSUE #2346: Add bottom margin to event tags in event card
**Describe the bug**

The tags in event cards shown on homepage are quite close to the bottom boundary of the card. It will be better to add a bottom margin.
Also in the responsive mode, the tags go out of cards.
**To Reproduce**
1. Go to '/'
2. See error

**Expected behavior**

There should be some gap between tags and bottom boundary of card
**Screenshots**

Screenshot from 2019-03-20 15-47-08

Screenshot from 2019-03-20 16-12-58

**Additional context**

I am working on it.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This fixes the event card responsiveness. Currently the tags are too much near the bottom boundary of the card and in responsive view, it crosses the boundary.

#### Changes proposed in this pull request:

- Remove the class with hardcoded height of 355px

## Screenshots
Screenshot from 2019-03-20 16-12-43
Screenshot from 2019-03-20 16-12-47

Fixes #2346

### CODE REVIEW

**Comment by mohitm15** [comment]
@Anupam-dagar But this is not showing in my laptop .... even after checking the responsiveness. I attached a screen shot too.

when screen is resized
Screenshot from 2019-03-20 18-08-52

**Comment by Anupam-dagar** [comment]
@mohitm15 did you fetch the PR properly, it is coming correct for me for all screens.

**Comment by mohitm15** [comment]
No what l am saying that it is even coming correct before fetching your changes...

**Comment by Anupam-dagar** [comment]
@mohitm15 The issue is reproducible, check https://github.com/fossasia/open-event-frontend/issues/2346

**Comment by CosmicCoder96** [comment]
This is not a valid fix. It actually worsens the UI. If you remove the class which fixed the width, the cards will have different heights when the content of one is longer.
Screenshot from 2019-03-22 10-22-29

**Comment by mohitm15** [comment]
@anupam-dagar this is what I'm talking about as mentioned by CosmiCoder96

**Comment by kushthedude** [comment]
@Anupam-dagar Can i make it fix or you want to try it again because I think i have got a solution ?

**Comment by CosmicCoder96** [comment]
@kushthedude there are plenty of other open issues available, allow a contributor minimum of 5 days to work on them before asking to take it over. Thanks!

**Comment by Anupam-dagar** [comment]
@kushthedude I was travelling, I have the fix for this, I'll push by night

**Comment by kushthedude** [comment]
Thats great

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have done the required changes. I have done some refactoring of code which makes the code simpler than the earlier version. The event cards for grid now use the method as defined by semantic ui rather than us creating a grid explicity and managing the responsiveness via `if` conditions.

**Comment by kushthedude** [comment]
@Anupam-dagar Your test is failing update the tests of event cards

File: app/styles/partials/event-card.scss [hunk 1]
```diff
@@ -36,3 +36,8 @@
     height: 130px;
   }
 }
+
+.display-block {
+  display: block;
+}
+
```
Comment (kushthedude) [inline app/styles/partials/event-card.scss]: @Anupam-dagar If you just fix the .event.card to 365 px in this file, Your work will be done and all this much wont be needed .

Comment (Anupam-dagar) [reply app/styles/partials/event-card.scss]: @kushthedude I know that. But doing that will be just assuming the max height of the card. Also I did a code refactor, this involves less conditional decision making via `if` and explicit definition for mobile version. You should take a look here once https://semantic-ui.com/views/card.html.
Just for the pointer, following will happen if I put the height to 365px, it must be atleast 380px.
Screenshot from 2019-03-27 01-40-49

Comment (kushthedude) [reply app/styles/partials/event-card.scss]: I think then you need to add a test file for the file you added .

Comment (Anupam-dagar) [reply app/styles/partials/event-card.scss]: @kushthedude I have fixed it :)

Comment (kushthedude) [reply app/styles/partials/event-card.scss]: Great lets wait for Travis Review 😄 👍

**Comment by Anupam-dagar** [comment]
I guess then assuming a maximum height and changing the height property should be okay with this. I'll look and update accordingly.

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have updated my PR.

**Comment by CosmicCoder96** [comment]
This increases the height of the overall card, not the tags area. And also after some searching -- this will get automatically resolved soon when ember-semantic-ui will support semantic-ui 2.4 in it's next release. We can come back to this then.

**Pull request closed without merging**